tensorflow/core/kernels/mkl/mkl_fused_instance_norm_op.cc

```diff
@@ -13,7 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#ifdef INTEL_MKL
+#if defined(INTEL_MKL) && !defined(ENABLE_ONEDNN_V3)
```

Comment:
Could you please add `&& !defined(ENABLE_ONEDNN_V3)` in the comment at the `#endif` line at the end of the file as well?

Reply:
@penpornk Looks like this kernel got disabled for oneDNN v3.x after rebasing with master. I have fixed it now. Please take a look.

Reply:
Got it. Thank you!